projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js

```diff
@@ -8,8 +8,9 @@ import {
 	Modal,
 	TextControl,
 	Icon,
-	SelectControl,
-	ButtonGroup,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	__experimentalToggleGroupControlOptionIcon as ToggleGroupControlOptionIcon,
```

Comment:
To use those experimental APIs, you would need to add linting exceptions:

```suggestion
	__experimentalToggleGroupControl as ToggleGroupControl, // eslint-disable-line @wordpress/no-unsafe-wp-apis
	__experimentalToggleGroupControlOption as ToggleGroupControlOption, // eslint-disable-line @wordpress/no-unsafe-wp-apis
	__experimentalToggleGroupControlOptionIcon as ToggleGroupControlOptionIcon, // eslint-disable-line @wordpress/no-unsafe-wp-apis
```

Since those may not be available depending on the site's version of Gutenberg / core installed, I would recommend adding a fallback to ensure no errors, when `ToggleGroupControl`, `ToggleGroupControlOption`, or `ToggleGroupControlOptionIcon` do not exist.